### PR TITLE
changed query parameter to support new query object syntax

### PR DESCRIPTION
**Pull request by shrynx**
Nothing much here, just changed the loaders syntax from query parameters to query object supported by webpack 2.

### CODE REVIEW

File: internals/webpack/webpack.base.babel.js [hunk 1]
```diff
@@ -31,10 +31,19 @@ module.exports = (options) => ({
       loader: 'file-loader',
     }, {
       test: /\.(jpg|png|gif)$/,
-      loaders: [
-        'file-loader',
-        'image-webpack?{progressive:true, optimizationLevel: 7, interlaced: false, pngquant:{quality: "65-90", speed: 4}}',
-      ],
+      loaders: 'file-loader',
+    }, {
+      test: /\.(jpg|png|gif)$/,
```
Comment (samit4me) [inline internals/webpack/webpack.base.babel.js]: This file pattern match is now listed twice?
This also seems to differ from the [official documentation](https://github.com/tcoopman/image-webpack-loader#usage), is there a reason for this?

**Comment by shrynx** [comment]
I was copying some features from this repo into my own webpack config and noticed these changes. It just makes the query parameter little more readable instead of the get like syntax, this is a webpack 2 feature https://gist.github.com/sokra/27b24881210b56bbaff7#loaders-configuration and https://medium.com/modus-create-front-end-development/webpack-2-tree-shaking-configuration-9f1de90f3233#.eqv4l9o8h (first point). 
Thanks for pointing out the duplicity, i have made the required change.
```js
test: /\.(jpg|png|gif)$/,
      loaders: [
        'file-loader',
        {
          loader: 'image-webpack',
          query: {
            progressive: true,
            optimizationLevel: 7,
            interlaced: false,
            pngquant: {
              quality: '65-90',
              speed: 4,
            },
          },
        },
      ],
```
Sorry i missed the remark in contribution guide.

**Comment by shrynx** [comment]
if you wish i can update with necessary changes and make a new pull request.

**Comment by samit4me** [comment]
Yes, unfortunately, the PR will need to be on the dev branch. I have not tested your code, but looking at the docs for `image-webpack` it would seem that the `image-webpack` loader needs to be chained with `file-loader`, which does not seem to be the case here. If you wouldn't mind testing this out before resubmitting the PR, that would be greatly appreciated 👍

**Comment by samit4me** [comment]
Closing as there is a new PR #1198

**Pull request closed without merging**